doc/topics/cloud/qs.rst

```diff
@@ -0,0 +1,128 @@
+.. _salt-cloud-qs:
+
+=====================
+Salt Cloud Quickstart
+=====================
+
+Salt Cloud is built-in to Salt and is configured on and executed from your Salt
+Master. On some platforms, Salt Cloud is installed by the ``salt-cloud``
+package.
+
+This quickstart walks you through the basic steps of setting up a cloud host
+and defining some virtual machines to create.
+
+Define a Provider
+-----------------
+The first step is to add the credentials for your cloud host. Credentials
+and other settings provided by the cloud host are stored in provider configuration files.
+Provider configurations contain the details needed to connect to a cloud host such as EC2, GCE, Rackspace, etc.,
+and any global options that you want set on your cloud minions (such as the location of your Salt Master).
+
+On your Salt Master, browse to ``/etc/salt/cloud.providers.d/`` and create a file called ``<provider>.provider.conf``,
```

Comment:
Remove the `provider` part from `<provider>.provider.conf`. Should just be `<provider-name>.conf`or something like that.